src/sorts/distribute/GuessSort.java

```diff
@@ -1,64 +1,56 @@
 package sorts.distribute;
 
 import main.ArrayVisualizer;
-import sorts.templates.Sort;
+import sorts.templates.BogoSorting;
 
-public final class GuessSort extends Sort {
-    private final double DELAY = 0.0001;   // trust me, this shouldn't be greater than tiny!
+public final class GuessSort extends BogoSorting {
     public GuessSort(ArrayVisualizer arrayVisualizer) {
         super(arrayVisualizer);
+
         this.setSortListName("Guess");
         this.setRunAllSortsName("Guess Sort");
         this.setRunSortName("Guess Sort");
         this.setCategory("Impractical Sorts");
-        this.setComparisonBased(true);
+        this.setComparisonBased(false);
         this.setBucketSort(false);
         this.setRadixSort(false);
         this.setUnreasonablySlow(true);
         this.setUnreasonableLimit(8);
         this.setBogoSort(false);
     }
-    
+
     // PROGRAMMER'S NOTE: This sort is intentionally bad, it is purposefully un-optimized.
+    // OTHER PROGRAMMER'S NOTE: haha too bad this isn't even the same sort anymore
```

Comment:
Did you completely remove _fluffyy's original Guess Sort?

Reply:
Forgot to mention it was moved to DeterministicGuessSort.java (and GuessSort.java is the new random version), sorry!

Oops, also forgot to add documentation

Reply:
It appears that you changed that one too though. [src/sorts/distribute/DeterministicGuessSort.java:23](https://github.com/Gaming32/ArrayV-v4.0/pull/34/files#diff-1532c67b420cc94da7cb1a97e1d650ef34469c91530a4f1895fa9cc3806ef4feR23)

Reply:
Yes, I optimized it >:)

Reply:
But it's not the original anymore then. The original was "__Purposefully__ unoptimized."

Reply:
but for no reason

Reply:
There was a reason. The goal was to make the worst-possible deterministic sort.

Reply:
true, but I personally view Guess Sort as "the sort that guesses indices" and don't see much appeal in adding additional un-optimizations

Reply:
I see your point, but I feel you should still have the original in there somewhere.

Reply:
How about GuessSort.java? The random version could be called RandomGuessSort.java

Reply:
alright, what about "OptimizedGuessSort.java" for optimized deterministic? (though I don't think it's optimized enough to determine such a title, any other ideas?)

Reply:
I like "OptimizedGuessSort.java" (I don't really have any other ideas)